lib/solidus_comments.rb

```diff
@@ -1,9 +1,8 @@
 # frozen_string_literal: true
 
-require 'spree_api'
-require 'spree_backend'
 require 'solidus_core'
 require 'solidus_support'
 require 'acts_as_commentable'
+
+require 'solidus_comments/version'
 require 'solidus_comments/engine'
-require 'deface'
```

Comment:
If the extension has overrides, which it does, you need to require Deface for the overrides to work.

Reply:
I checked if deface works using the dummy app and it does. WDYT?

Reply:
Interesting... I've had this problem in the past, where removing `require 'deface'` would not display the overrides. After all, if you're not requiring Deface and no one else is, the gem cannot be loaded. Could you check if maybe something else is loading Deface in your tests?

Reply:
![image](https://user-images.githubusercontent.com/9986708/72617252-dbfd8c00-3938-11ea-866d-9a217e3c3aa4.png)

It seems that anyone require it. 
Maybe we can require it avoiding any trouble...

Reply:
Yeah, let's do that, because otherwise it's not clear how the overrides would be loaded.

Reply:
could it be required by another extension? Maybe `solidus_auth_devise`?